build.ps1

```diff
@@ -251,9 +251,9 @@ function PushPackage {
     }
 
     if($DryRun){
-        Write-Host "Dry run enabled, not pushing package."
+        Write-Info "Dry run enabled, not pushing package."
     } else {
-        Write-Host "Pushing $PackageFile to $EdFiNuGetFeed"
+        Write-Info ("Pushing $PackageFile to $EdFiNuGetFeed")
```

Comment:
The use of parenthesis on PowerShell function calls is not required, and in some cases can actually be a problem. I'll test to see if this works.

Reply:
It works fine. We can leave it alone.